bedrock/firefox/templates/firefox/nothing-personal/index.html

```diff
@@ -149,89 +194,55 @@ <h4>You’re great — probably.</h4>
-          </div>
-
-          <div class="c-sticky-note c-attached-sticky">
-            <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
-            <br>
-            <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
-            <br>
-            <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
-
-            {{ apple_app_store_button(href=ios_url, class_name='app-store-badge') }}
-            {{ google_play_button(href=android_url, class_name='play-store-badge') }}
-
-            <p>Ok, bye.</p>
-          </div>
-
         </div>
+        {% call browser_border(id="newsletter-signup", class="c-newsletter", heading='Newsletter') %}
+          <h4 class="c-mono-title"><span aria-hidden="true">[</span>Sign up for our newsletter<span aria-hidden="true">]</span></h4>
+
+          {{ email_newsletter_form(
+            include_title=False,
+            include_language=False,
+            include_country=False,
+            newsletters='mozilla-and-you, nothing-personal-college-interest-waitlist',
+            multi_opt_in_required=True
+          )}}
+
+          <p>By subscribing, you’ll receive Mozilla updates, plus exclusive college-related content if you opt in. You may unsubscribe from either at any time.</p>
         {% endcall %}
-
-         {% call browser_border(id="newsletter-signup", class='mzp-l-content mzp-t-content-md mzp-t-content-nospace', heading='Newsletter') %}
-          <div class="c-browser-window-top">
-            <h4><span aria-hidden="true">[</span>Sign up for our newsletter<span aria-hidden="true">]</span></h4>
-
-            {{ email_newsletter_form(
-              include_title=False,
-              include_language=False,
-              include_country=False,
-              newsletters='mozilla-and-you, nothing-personal-college-interest-waitlist',
-              multi_opt_in_required=True
-            )}}
-
-            <p>By subscribing, you’ll receive Mozilla updates, plus exclusive college-related content if you opt in. You may unsubscribe from either at any time.</p>
+      </div>
+      <div class="c-slide-in-container">
+        <div class="c-sticky-note" data-animation="slide-in">
+          <span aria-hidden="true">>>></span> {{ download_firefox_thanks(alt_copy='download', button_class='mzp-t-primary mzp-t-lg c-download-firefox-cta', download_location='secondary cta') }} <span aria-hidden="true"> <<<< </span>
```

Comment:
It has always bothered me that this is `>>>>` not `&gt;&gt;&gt;&gt;` Can you change them since you're in there 🙏